library/src/test/java/org/apache/apex/malhar/lib/wal/FileSystemWALTest.java

```diff
@@ -53,20 +55,33 @@ private static Slice getRandomSlice(int len)
   {
     private String targetDir;
     FileSystemWAL fsWAL = new FileSystemWAL();
+    Configuration conf = new Configuration();
+    FileSystem fs;
 
     @Override
     protected void starting(Description description)
     {
-      TestUtils.deleteTargetTestClassFolder(description);
       targetDir = "target/" + description.getClassName() + "/" + description.getMethodName();
+
+      try {
```

Comment:
If you refer to tests that specifically test behavior that relies on HDFS then yes (it is an integration test), but expecting every other unit test to use Hadoop API is certainly not appropriate.